flake8_trio.py

```diff
@@ -612,17 +611,18 @@ def visit_For(self, node: ast.For | ast.While):
         if not self.unraised:
             return
 
+        # the following block is duplicated in Visitor107_108
         infinite_loop = False
         if isinstance(node, ast.While):
             try:
                 infinite_loop = body_guaranteed_once = bool(ast.literal_eval(node.test))
-            except Exception:
+            except ValueError:
```

Comment:
IIRC it's *possible* to get `TypeError` on some versions?  Could be wrong, but I'd leave it as `Exception` here and below because it's a small block and I don't really care what kind of exception this raises.

Reply:
sounds reasonable, fixed :+1: